config.toml

```diff
@@ -80,12 +80,14 @@ notAlternative = true
 [params]
   enable_search = true
   robots = ''
+  images = ["chef-progress-logo.svg"]
   menuOrder = ["overview", "automate", "compliance",  "desktop",  "infra", "habitat", "inspec", "workstation", "effortless", "release_notes", "legacy", "extra"]
 
 [taxonomies]
   category = "categories"
   tag = "tags"
   resource = "resources"
+  series = "series"
```

Comment:
Do we need these changes?

Reply:
Not in this PR...it is noise from something else I'm working on.